sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncMixedBenchmark.java

```diff
@@ -30,20 +31,30 @@ class AsyncMixedBenchmark extends AsyncBenchmark<Object> {
     @Override
     protected void performWorkload(BaseSubscriber<Object> documentBaseSubscriber, long i) throws InterruptedException {
         Flux<? extends Object> obs;
+
+        CosmosAsyncContainer containerToUse = cosmosAsyncContainer;
+
```

Comment:
This code block is repeated in every benchmark - why does the base class need to have 3 container instances - why not just initialize the "right" one on the base class?

Reply:
Would reduce the churn rate in the individual benchmarks